core/components/organisms/textField/TextFieldWithInput.tsx

```diff
@@ -0,0 +1,226 @@
+import * as React from 'react';
+import { Input, Label, HelpText, Text } from '@/index';
+import { IconProps } from '@/index.type';
+import { AutoComplete } from '@/common.type';
+import { BaseProps } from '@/utils/types';
+
+export type InputSize = 'tiny' | 'regular' | 'large';
+
+export interface TextFieldInputProps extends BaseProps {
```

Comment:
Instead of defining TextFieldInputProps separately please use the existing InputProps directly